docs/installingNative.md

```diff
@@ -63,6 +70,16 @@ Most of these prerequisites can be installed using the package manager of your O
     curl -fsSL https://d2lang.com/install.sh | sh -s --
     ```
 
+    The pgvector Postgres extension can be installed with the following script:
+
+    ```sh
+    cd /tmp
+    git clone --branch v0.8.0 https://github.com/pgvector/pgvector.git
+    cd pgvector
+    make
+    sudo make install
+    ```
```

Comment:
Should this be added to the macOS section too?

Reply:
Definitely, yeah. `brew install pgvector` should be sufficient.

Reply:
Done in 188d3fed94a26fb09b7df895e5cf33ad7af09908. I added it to the existing `brew install` command.